src/ring_theory/jacobson_ideal.lean

```diff
@@ -86,6 +96,133 @@ theorem mem_jacobson_iff {I : ideal R} {x : R} :
         neg_mul_eq_neg_mul_symm, neg_mul_eq_mul_neg, mul_comm x y]; exact M.mul_mem_right hy)
     (him hz)⟩
 
+theorem eq_jacobson_iff_Inf_maximal {I : ideal R} :
+  I.jacobson = I ↔ ∃ M ⊆ {J : ideal R | J.is_maximal ∨ J = ⊤}, I = Inf M :=
+begin
+  use λ hI, ⟨{J : ideal R | I ≤ J ∧ J.is_maximal}, ⟨λ _ hJ, or.inl hJ.right, hI.symm⟩⟩,
+  rintros ⟨M, hM, hInf⟩,
+  refine le_antisymm _ le_jacobson,
+  intros x hx,
+  rw hInf,
+  erw mem_Inf at ⊢ hx,
+  intros I hI,
+  cases hM hI with is_max is_top,
+  { refine hx ⟨le_Inf_iff.1 (le_of_eq hInf) I hI, is_max⟩ },
+  { rw is_top, exact submodule.mem_top }
+end
+
+lemma eq_jacobson_iff_not_mem {I : ideal R} :
+  I.jacobson = I ↔ ∀ x ∉ I, ∃ M : ideal R, (M ∈ {J : ideal R | I ≤ J ∧ J.is_maximal}) ∧ x ∉ M :=
```

Comment:
Again, I think just saying` I ≤ M ∧ M.is_maximal` is more readable than saying it's in the set of things with this property.